mtags/src/main/scala-3/scala/meta/internal/pc/completions/Completions.scala

```diff
@@ -237,7 +239,18 @@ class Completions(
           if Fuzzy.matches(td.symbol.name.decoded, filename) =>
         val values = FilenameCompletions.contribute(filename, td)
         (values, true)
-
+      case (lit @ Literal(Constant(str: String))) :: parent :: _ =>
```

Comment:
Could you add comment with example what will be matched by this case?

Reply:
Added!